src/main/app/form/Form.test.ts

```diff
@@ -23,19 +23,15 @@ describe('Form', () => {
           { label: l => l.dateFormat['month'], name: 'month' },
           { label: l => l.dateFormat['year'], name: 'year' },
         ],
-        parser: value => covertToDateObject('dateField', value),
-        validator: value => areFieldsFilledIn(value as CaseDate),
+        parser: value => covertToDateObject('dateField', value as Record<string, unknown>),
+        validator: value => areDateFieldsFilledIn(value as CaseDate),
       },
-      someCheckboxes: {
+      checkboxes: {
         type: 'checkboxes',
+        validator: isFieldFilledIn,
         values: [
-          { name: 'optionalCheckbox', label: () => 'optional', value: Checkbox.Checked },
-          {
-            name: 'requiredCheckbox',
-            label: () => 'required checkbox',
-            value: Checkbox.Checked,
-            validator: isFieldFilledIn,
```

Comment:
Could we keep a test for a required checkbox as there are a few places where the user is required to tick a checkbox.

Reply:
Unless checkbox individual checkbox validation is no longer a thing, and it doing it on the checkbox group?

Reply:
the method still works, i did move it outside the individual values but ill add it to the test as well.

Reply:
[line 31](https://github.com/hmcts/nfdiv-frontend/pull/446/files#diff-f1e9f6cea5d7aff8749494ca8672015defeb8afeaefd59eebb16d7222fa1fcbbR31)